.github/workflows/reusable-integration-tests-on-prem.yml

```diff
@@ -14,6 +14,7 @@ jobs:
   integration-tests:
     strategy:
       matrix:
+        node: [ "lts/*", 19 ]
```

Comment:
The changes in this file are only to test if it works!
will be removed when successfully tested

Reply:
Removed